OctoKit/PullRequest.swift

```diff
@@ -320,7 +397,27 @@ enum PullRequestRouter: JSONPostRouter {
                 parameters["base"] = base
             }
             if let mantainerCanModify = mantainerCanModify {
-                parameters["maintainer_can_modify"] = (mantainerCanModify ? "true" : "false")
+                parameters["maintainer_can_modify"] = mantainerCanModify
+            }
+            return parameters
+
+        case let .createPullRequest(_, _, _, title, head, headRepo, base, body, mantainerCanModify, draft):
+            var parameters: [String: Any] = [
+                "title": title,
+                "head": head,
+                "base": base
+            ]
+            if let headRepo {
```

Comment:
Tests are failing because in the swift version we support this shorthand is not available.